Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apt-get install -y fonts-liberation libappindicator3-1 libasound2 libatk-bri
 RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb && \
```

Comment:
I want to change the version here to fix it to a specific chrome version, but I can't find a version-pegged link to substitute instead of `https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb` -- hmm!

Reply:
Maybe https://chromereleases.googleblog.com/ or ah! Here!

https://www.chromium.org/getting-involved/download-chromium

Reply:
Hm. That says old versions are not provided as they lack security updates. It shows a pathway to finding old binaries but it's complicated and I wasn't able to find one for 75 on my first try. I feel like it's not a best practice to point at old chromes, but we're stuck as chromedriver isn't working.

Reply:
OK, so I'm stuck here, i can't find an old version for chrome 75-- so can't make this change:

```
    wget https://chromedriver.storage.googleapis.com/75.0.3770.140/chromedriver_linux64.zip && \
```

Do we just wait a bit for a solution?

Reply:
OK, found a chrome 74 to point to: https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Linux_x64/638880/

Reply:
Can we use https://www.googleapis.com/download/storage/v1/b/chromium-browser-snapshots/o/Linux_x64%2F638880%2Fchrome-linux.zip?generation=1552011881622744&alt=media ?

Reply:
Ok, trying the above link